pkg/kubectl/cmd/describe.go

```diff
@@ -64,6 +75,24 @@ func RunDescribe(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []s
 		return err
 	}
 
+	if len(args) == 0 {
+		fmt.Fprint(out, `
```

Comment:
This is an error message, so should be printing to stderr. Better still would be to roll it into the returned error's message.